Clarify dohpath manipulation attack (#76)
ietf-wg-add/draft-ietf-add-ddr: draft-ietf-add-ddr.md

DoH resolvers that allow discovery using DNS SVCB answers over unencrypted
DNS MUST NOT provide differentiated behavior based on the HTTP path alone,
since an attacker could modify the "dohpath" parameter. For example, if a

> DoH resolver provides provides a filtering service for on URI path, and

Reviewer: for _one_ URI path (typo)

Author: ```suggestion
DoH resolver provides provides a filtering service for one URI path, and
```

a non-filtered service for another URI path, an attacker could select

> which of these services is used by modifying the "dohpath" parameter. Such
> differentiated services need to also rely on other aspects of connections
> and URIs being different, such as using separate resolver IP addresses and
> hostnames.

Reviewer 2: I wonder if you need to be so definitive here.  A lot of networks are very confident in their ability to get DHCP to endpoints without these attacks.  Is it not enough to identify the risk and let people reach their own conclusions?

I mean, if it takes a backhoe and cable splicing to mount the attack and the outcome is that you get to switch a user from a filtered resolver to a unfiltered one (or vice versa), maybe that's something that doesn't require a massive service reconfiguration to counter.

So maybe you can instead say:

```suggestion
which of these services is used by modifying the "dohpath" parameter.
These attacks can be mitigated by providing separate resolver IP
addresses or hostnames.
```

Author: That works for me! Simpler is good here.
